db/changes.go

```diff
@@ -95,6 +96,11 @@ func (db *Database) addDocToChangeEntry(ctx context.Context, entry *ChangeEntry,
 		return
 	}
 
+	dbCollection, ok := db.CollectionByID[entry.CollectionID]
+	if !ok {
+		base.FatalfCtx(ctx, "Changes feed: could not determine collection from change entry")
```

Comment:
This can only be programmer error (forgetting to add collectionID to `ChangeEntry` or `LogEntry`, so I thought Fatalf might be appropriate (I even thought about panicking)

Reply:
Resolved with Warnf